cms/static/js/views/previous_video_upload.js

```diff
@@ -31,6 +37,31 @@ define(
                     this.template(_.extend({}, this.model.attributes, renderedAttributes))
                 );
                 return this;
+            },
+
+            removeVideo: function(event) {
+                var videoView = this;
+
+                event.preventDefault();
+
+                ViewUtils.confirmThenRunOperation(
+                    gettext('Remove Video Confirmation'),
+                    gettext('Are you sure you wish to remove this video. It cannot be reversed!'),
```

Comment:
@muhammad-ammar @sstack22 
In the interests of trying to keep word count down, I'm thinking that there's no need to say "this action cannot be reversed" given that we are explaining why removing an item from the list doesn't affect the video display in the course - do you agree?

My suggestion would be (please check for accuracy):

"Are you sure you want to remove this video from the list? Note: Removing a video from this list does not affect course content. Any content that uses a previously uploaded video ID continues to display in the course."